### ISSUE TITLE
How was weights/hardnet_petite_base.pth trained?

### ISSUE THREAD

**Issue by blake-varden** [PingoLH/FCHarDNet#36]
Hi, I'm looking to convert this repo to Tensorflow.  I believe most of the operations here are standard and it should be relatively straightforward to convert to Tensorflow.

I will however need to regenerate the pretrained weights for hardnet_petite_base.  I was wondering if you could share the details of how you trained this?

Is the structure, just the decoder portion of the network followed by an FC layer to the number of classes and then Trained on ImageNet for 100 epochs?

**Comment by PingoLH** [comment]
Hi, yes we trained it with ImageNet for the first 100 epochs out of a totally 150 epochs of cosine learning rate schedule.

**Comment by blake-varden** [comment]
Hi thank you. And to confirm the architecture of the petitie base is : the decoder + fc layer + softmax  to classify the classes?

**Comment by PingoLH** [comment]
Yes, just add a fc layer with a cross entropy loss should be fine.

**Comment by yoondonghwee** [comment]
How much hours have you spent for training the model with ImageNet?
